Resources/AzureDevOps/JobTemplates/BuildAndPublishLinuxContainers.yml

```diff
@@ -0,0 +1,14 @@
+parameters:
```

Comment:
Suspect these need to be pushed thru to the 'BuildAndPublishContainers' step and the parameters added there too